app/routes/projects/index.tsx

```diff
@@ -42,7 +42,7 @@ const FACETS = [
   "status",
   "skill",
   "label",
-  "disciplines",
+  "discipline",
```

Comment:
This fixed a bug on the search page, it was not searching on the "Looking for:" filter.